Pod/Classes/PINRemoteImageManager.h

```diff
@@ -424,6 +424,17 @@ typedef void(^PINRemoteImageManagerProgressDownload)(int64_t completedBytes, int
                              progressImage:(nullable PINRemoteImageManagerImageCompletion)progressImage
                                 completion:(nullable PINRemoteImageManagerImageCompletion)completion;
 
+
+/**
+ Adds an image manually into the memory and disk cache.
+ 
+ @param data NSData with the raw image data.
+ @param url NSURL where the image resides.
+ @processorKey NSString key to uniquely identify processor and process. Will be used for caching processed images.
+ @additionalCost NSUInteger The cache cost of the object.
+ */
+- (BOOL) insertImageDataIntoCache:(nonnull NSData*)data withURL:(nonnull NSURL *)url processorKey:(nullable NSString *)processorKey additionalCost: (NSUInteger) additionalCost;
```

Comment:
Ok, lets go with additionalCost, can you update the comment to say `@additionalCost NSUInteger the additional cost (for cache evicting purposes) to generate the processed image`